tests/vmi_configuration_test.go

```diff
@@ -2702,7 +2702,7 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				pinnedCPUsList, err := hw_utils.ParseCPUSetLine(output, 100)
 				Expect(err).ToNot(HaveOccurred())
 
-				output, err = tests.ListCgroupThreads(readyPod)
+				output, err = listCgroupThreads(readyPod)
```

Comment:
Maybe call to `exec.ExecuteCommandOnPod` once and inline it here? this function is not called in other places.

Reply:
I prefer not to. using function improves readability. I don't want to make long function longer.